DashAI/back/job_queues/simple_job_queue.py

```diff
@@ -0,0 +1,96 @@
+import uuid
+from asyncio import Queue
+from typing import Any, Coroutine, List
+
+from DashAI.back.job_queues.base_job_queue import BaseJobQueue, Job, JobQueueError
+
+
+class SimpleJobQueue(BaseJobQueue):
+    """JobQueue implementation using asyncio Queue."""
+
+    queue: Queue = Queue()
+
+    def _search_and_split(self, job_id: int) -> tuple[List[Job], Job, List[Job]]:
```

Comment:
Use Tuple to ensure python 3.8 compatibility.

```suggestion
    def _search_and_split(self, job_id: int) -> Tuple[List[Job], Job, List[Job]]:
```